.github/workflows/rebuildDependencies.yml

```diff
@@ -134,20 +135,28 @@ jobs:
         git sparse-checkout set \
           recipes/minizip \
           recipes/flac \
+          recipes/onnx \
+          recipes/onnxruntime \
           $recipePathQt \
 
         git checkout
 
+        # Patch conan recipe to prevent dragging libcurl as a dependency
+        # (can't be done via VCMI's conanfile as onnxruntime is built beforehand)
+        git apply --ignore-whitespace ../conan_patches/onnxruntime/recipe.diff
+
         # versions must be synced with: conan_patches/<package>/conandata.yml
         # if no custom patches are required for a package, it should be removed from here
-        for p in minizip/1.3.1 flac/1.4.2 ; do
+        for p in minizip/1.3.1 flac/1.4.2 onnx/1.16.2 onnxruntime/1.18.1 ; do
           IFS_OLD="$IFS"
           IFS=/
           read package version <<<"$p"
           IFS="$IFS_OLD"
 
           if [[ $package == qt ]] ; then
             packagePath="$recipePathQt/5.x.x"
+          elif [[ $package =~ ^onnx ]] && ${{ contains(matrix.conan_options, 'with_onnxruntime=False') }} ; then
```

Comment:
but I don't see such option in the matrix?

Reply:
Should I remove the `qt` check then as well?